GliaWidgets/Theme/Survey/Theme+Survey.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+import UIKit
+
+public extension Theme {
```

Comment:
I think it is better to be more explicit with access modifiers by avoiding adding it to extension, and add it to declaration of a type instead:
```
 extension Theme {
    ...

    /// Text style.
    public struct Text {
        ...
```

Reply:
fixed 2720cc15d186e3ab1ba91d1d759f26515e32142b